Bottom: ffbbb287b1a2e4fbdbcedb90103f988acb8fcb42
Top:    f8e50db6a6df431f252c42108acad59c82371cf7
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2013-08-27 11:38:40 -0700

Refresh of a

---

diff --git a/examples/.gitignore b/examples/.gitignore
index ed17494..ce9156d 100644
--- a/examples/.gitignore
+++ b/examples/.gitignore
@@ -8,6 +8,7 @@
 /.libs
 /.dirstamp
 
+/cmtime
 /mckey
 /rcopy
 /rdma_client
